app/models/order_balance.rb

```diff
@@ -24,7 +24,7 @@ def to_f
   end
 
   def +(other)
-    to_f + other.to_f
+    amount + other.to_f
```

Comment:
If we go with this renaming, we need to call `other.amount` too

Reply:
But... what if we're adding something that's not an `OrderBalance` object? Like `order.outstanding_balance + shipping_fee` or something. It'll give unexpected results, right? If we're adding two objects that are both an `OrderBalance`, when we do `other.to_f` it'll already be delegated to `#amount` by default, right?

Reply:
you are right. I wasn't thinking `other` can be of any type, not just `OrderBalance` :see_no_evil: